src/test/java/de/tum/in/www1/artemis/FileUploadSubmissionIntegrationTest.java

```diff
@@ -356,6 +360,22 @@ public void getDataForFileUpload() throws Exception {
         assertThat(submission.getLatestResult().getFeedbacks().size()).as("No feedback should be returned for editor").isEqualTo(0);
     }
 
+    @Test
+    @WithMockUser(value = "student1")
+    public void getDataForFileUpload_latestSubmissionOfParticipationNull() throws Exception {
+        FileUploadSubmission fileUploadSubmission = ModelFactory.generateFileUploadSubmission(true);
+        List<Feedback> feedbacks = ModelFactory.generateFeedback();
+        fileUploadSubmission = database.saveFileUploadSubmissionWithResultAndAssessorFeedback(releasedFileUploadExercise, fileUploadSubmission, "student1", "tutor1", feedbacks);
+        database.updateExerciseDueDate(releasedFileUploadExercise.getId(), ZonedDateTime.now().minusHours(1));
+        Participation participation = fileUploadSubmission.getParticipation();
+        participation.setResults(null);
+        FileUploadSubmission submission = request.get("/api/participations/" + participation.getId() + "/file-upload-editor", HttpStatus.OK, FileUploadSubmission.class);
+        assertThat(submission).isNotNull();
+        assertThat(submission.getLatestResult()).isNotNull();
+        assertThat(submission.isSubmitted()).isTrue();
+        assertThat(submission.getLatestResult().getFeedbacks().size()).as("No feedback should be returned for editor").isEqualTo(0);
```

Comment:
you could also assert ,that the assessor information is not included in the response right? I think this should be covered by regression tests.